dev-python/spyder/spyder-4.2.1.ebuild

```diff
@@ -0,0 +1,230 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8} )
+
+inherit optfeature xdg distutils-r1
```

Comment:
```
 * DISTUTILS_USE_SETUPTOOLS value is probably incorrect
 *   have:     DISTUTILS_USE_SETUPTOOLS=bdepend (or unset)
 *   expected: DISTUTILS_USE_SETUPTOOLS=no
```
Could you verify?

Reply:
This warning has been confusing me for quite some time now, it is clearly a false positive since `setup.py` has:
```
# Third party imports
from setuptools import setup, find_packages
from setuptools.command.install import install
```
but I do not understand why the eclass's QA check does not work correctly here.

Reply:
https://blogs.gentoo.org/mgorny/2020/10/21/distutils_use_setuptools-qa-spam-and-more-qa-spam/ Yeah, might be due to python-3.8 changes regarding setuptools. I wouldn't know any better.